components/AdaptiveTextInput.tsx

```diff
@@ -0,0 +1,39 @@
+import { TextInput, TextInputProps, StyleSheet } from 'react-native';
+import { useState } from 'react';
+import { View } from './Themed';
+import {
+    useFonts,
+    B612Mono_400Regular,
+    B612Mono_400Regular_Italic,
+    B612Mono_700Bold,
+    B612Mono_700Bold_Italic
+} from '@expo-google-fonts/b612-mono';
+
+type AdaptiveTextInputProps = TextInputProps & {
+    /**
+     * For default fontSize, charWidth = 9.5 is best. If initializing with custom fontSize
+     * style, you gotta figure out the right charWidth yourself. Good luck.
+     */
+    charWidth: number,
+};
+
+export default function AdaptiveTextInput(props: AdaptiveTextInputProps) {
+    const { style, value, ...otherProps } = props;
+    const [ fontLoaded ] = useFonts({B612Mono_700Bold});
+
+    if (!fontLoaded) {
+        return <View></View>
+    }
+
+    return (
+        <>
```

Comment:
The <> is not necessary as the <TextInput> can be the root component.

Reply:
Fixed